src/MfaVerifyController.js

```diff
@@ -133,10 +133,10 @@ export default BaseLoginController.extend({
       }
     });
 
-    if (!this.settings.get('features.hideSignOutLinkInMFA') &&
-        !this.settings.get('features.mfaOnlyFlow')) {
-      this.add(new FooterSignout(this.toJSON()));
-    }
+    this.add(new FooterMFA(_.extend(this.toJSON(), {
```

Comment:
Since the FooterMFA is only meant for MFA page, we can move `!this.settings.get('features.hideSignOutLinkInMFA') &&
         !this.settings.get('features.mfaOnlyFlow')` logic inside `FooterMFA` ?

Reply:
Done